app/views/all_trips/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<p class="title"><%= t(".title") %></p>
```

Comment:
A suggestion to use `h1` rather than `<p class="title">`

Reply:
Done in sha: `63a40af`